src/components/Home.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FiSearch } from 'react-icons/fi';
+import { SiWorldhealthorganization } from 'react-icons/si';
+import { getCountryData } from '../redux/country/countrySlice';
+
+function Home() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getCountryData());
+  }, []);
+
+  const countries = useSelector((store) => store.country);
+  const [querry, setQuerry] = useState('');
+  const dataSearch = (str) => setQuerry(str);
+
+  return (
+    <section>
+      <div className="search__container">
+        <span>
+          <FiSearch />
+        </span>
+        <input className="country__search" placeholder="Eg: Afghanistan , South Africa" onChange={(e) => dataSearch(e.target.value)} />
+      </div>
+      <div>
+        <h1>STATS BY COUNTRY</h1>
+      </div>
+      <div>
+        <ul className="container">
+          {
+      countries.covidData.filter((value) => value.country
+        .toLowerCase().includes(querry
+          .toLocaleLowerCase())).map((data) => (
+            <NavLink
+              to={`/details/${data.country}`}
+              key={data.country}
+              state={data.country === null ? '' : { country: data.country, flag: data.flag }}
+            >
+              <div className="country__info">
+                <div>
+                  <h2 className="country__name">{data.country}</h2>
+                  <div className="country__facts">
+                    <p className="country__population">{data.cases}</p>
+                    <p className="country__continent">{data.continent}</p>
+                  </div>
+                </div>
+                <SiWorldhealthorganization className="world-icon" />
+              </div>
+            </NavLink>
```

Comment:
- [ ] Great job for the functionality and the design but I have small notes so you can improve more as I saw in your design the cards for the countries is under each others but in the original design each two cards is beside each others so kindly make your design like the given design since it is one of the requirements to follow the design layout 💯🤩💪🏻🔥🔥

![image](https://user-images.githubusercontent.com/58553711/225319359-7e095f17-fe1a-4684-aa5b-ca86f5887a4b.png)
- [ ]  PLEASE look at the lines I drawed the cards is beside each other in the given design but yours under each others 🙏🏻 

- [ ]  I think You are missing the header in the home page the section between the nav and the cards kindly add it 💯
- Your design.
![image](https://user-images.githubusercontent.com/58553711/225319586-53d9daa7-4c1f-410e-aaf1-821c8312bfa7.png)


- Given design
![image](https://user-images.githubusercontent.com/58553711/225319700-1e3d039e-af2d-4dd9-b98e-ada572adcb4e.png)